src/sqlalchemy_cratedb/compiler.py

```diff
@@ -323,3 +324,36 @@ def for_update_clause(self, select, **kw):
         warnings.warn("CrateDB does not support the 'INSERT ... FOR UPDATE' clause, "
                       "it will be omitted when generating SQL statements.")
         return ''
+
+
+# TODO: There are certainly more to add here than just `object`?
+CRATEDB_RESERVED_WORDS = ["object"]
```

Comment:
What about `index`?

Reply:
Here is the list of CRATEDB keywords currently (master):
```
CROSS, CURRENT_DATE, INTERSECT, ELSE, END, EXCEPT, USING, CASE, AND, CURRENT_SCHEMA, ANY, ALL, SET, LIMIT, INPUT, NATURAL, CAST, DIRECTORY, IS, WHEN, IF, TABLE, RIGHT, OUTER, FULL, ORDER, SELECT, JOIN, ADD, SESSION_USER, CURRENT_TIME, GRANT, TRUE, LEFT, INTO, TRY_CAST, CURRENT_ROLE, INSERT, SOME, EXISTS, UPDATE, FALSE, CREATE, RESET, OFFSET, OBJECT, TRANSIENT, CURRENT_USER, IN, OR, FOR, ALTER, ASC, FUNCTION, NULL, FROM, DEFAULT, NOT, LIKE, UNION, DISTINCT, NULLS, HAVING, INNER, BY, PERSISTENT, STRATIFY, ARRAY, REVOKE, MATCH, DROP, ESCAPE, WHERE, COSTS, WITH, GROUP, INDEX, DELETE, COLUMN, ON, UNBOUNDED, RETURNS, THEN, LAST, USER, CALLED, RECURSIVE, BETWEEN, DESCRIBE, AS, EXTRACT, CURRENT_TIMESTAMP, DENY, FIRST, CONSTRAINT, DESC
```

Reply:
Thanks. I've established this list of keywords manually, and extended the test case to also use another reserved column, `index`.